backend/tests/unit/test_conflict_notification.py

```
@@ -0,0 +1,244 @@
+
+# ============================================================================
+# Test: notify_conflict_detected
+# ============================================================================
+
+
+class TestNotifyConflictDetected:
+    """Tests for notify_conflict_detected() method."""
+
+    @patch("backend.src.services.notification_service.NotificationService.deliver_push")
+    def test_sends_notification_to_enabled_user(
+        self, mock_deliver, notification_service, enabled_user, test_team,
```

Ruff (0.15.0)

[warning] Unused method argument: `enabled_user`

(ARG002)